doc/start/hardware-recommendations.rst

```diff
@@ -214,9 +224,12 @@ your CephFS metadata pool that points only to a host's SSD storage media. See
 Controllers
 -----------
 
-Disk controllers also have a significant impact on write throughput. Carefully,
-consider your selection of disk controllers to ensure that they do not create
-a performance bottleneck.
+Disk controllers (HBAs) can have a significant impact on write throughput.
+Carefully consider your selection to ensure that they do not create
+a performance bottleneck.  Notably RAID-mode (IR) HBAs may exhibit higher
+latency than simpler "JBOD" (IT) mode HBAs, and the RAID SoC, write cache,
+and battery backup can substantially increase hardware and maintenance
+costs.  Some RAID HBAs can be configured with an IT mode "personality".
```

Comment:
s/IT mode/IT-mode/

I think this is right, given that it's a phrasal adjective.

Reply:
sure